src/foam/core/AbstractDatePropertyInfo.java

```diff
@@ -85,4 +85,43 @@ public void updateSignature(FObject obj, Signature sig) throws SignatureExceptio
     long val = date.getTime();
     sig.update((ByteBuffer) bb.get().putLong(val).flip());
   }
+
+  protected abstract java.util.Date get_(Object o);
+  protected abstract java.util.Date cast(Object key);
+    
+//  public foam.lib.parse.Parser jsonParser() {
```

Comment:
Is this a TODO?

Reply:
Yes. Actually, it generate 10 failures. But, we may fix it.